OCast/src/test/java/org/ocast/core/models/OCastTest.kt

```diff
@@ -50,9 +51,9 @@ class OCastTest {
         """.trimIndent()
 
         // When
-        val deviceLayer = OCastRawDeviceLayer.decode(data)
-        val ocastData = OCastRawDataLayer.decode(deviceLayer.message.data)
-        val webAppConnectedStatus = OCastDataLayer.decode<WebAppConnectedStatus>(ocastData.params)
+        val deviceLayer = JsonTools.decode<OCastRawDeviceLayer>(data)
+        val ocastData = JsonTools.decode<OCastRawDataLayer>(deviceLayer.message.data)
```

Comment:
This is not a modification from this MR, but this is the opportunity to rename `ocastData` to `oCastData` (and `ocastMessage` to `oCastMessage`)